manifest.php

```diff
@@ -34,12 +34,12 @@
     'label' => 'Test core extension',
     'description' => 'TAO Tests extension contains the abstraction of the test-runners, but requires an implementation in order to be able to run tests',
     'license' => 'GPL-2.0',
-    'version' => '6.3.1',
+    'version' => '6.4.0',
     'author' => 'Open Assessment Technologies, CRP Henri Tudor',
     'requires' => array(
         'taoItems' => '>=2.20.1',
         'taoBackOffice' => '>=0.8',
-        'tao' => '>=10.1.0'
+        'tao' => '>=10.28.0'
```

Comment:
the updated communicator is not a "strong" requirement and taoTest will continue to work with or without it, but tao 10.28.0 is because it contains the `core/connectivity` module. The compatibility is increased.
I can update the dependency, but I though it wasn't required